runtime/src/iree/hal/drivers/cuda/cuda_driver.c

```diff
@@ -46,6 +46,26 @@ IREE_API_EXPORT void iree_hal_cuda_driver_options_initialize(
     iree_hal_cuda_driver_options_t* out_options) {
   memset(out_options, 0, sizeof(*out_options));
   out_options->default_device_index = 0;
+  out_options->default_rank = 0;
+  out_options->default_count = 0;
+}
+
+// Get the MPI world sizefrom the environmental variable. Return 0 if the
+// variable is not set.
+iree_status_t iree_hal_cuda_get_mpi_comm_world_size_from_env(int32_t* size) {
```

Comment:
```suggestion
static iree_status_t iree_hal_cuda_get_mpi_comm_world_size_from_env(int32_t* out_size) {
```
(out_* prefix as above)
(functions local to a file should be declared as static)